.gitignore

```diff
@@ -0,0 +1 @@
+autoencoder/MNIST_data/*
```

Comment:
If you are adding .gitignore cosider adding *.pyc there and remove the pyc files from this PR (and squash it, so the never make it to the repo).